packages/driver/types/internal-types.d.ts

```diff
@@ -99,3 +99,7 @@ type AliasedRequest = {
   alias: string
   request: any
 }
+
+// utility types
+type Omit<T, K extends keyof T> = Pick<T, Exclude<keyof T, K>>
```

Comment:
Looks like [Omit](https://www.typescriptlang.org/docs/handbook/utility-types.html#omittype-keys) was added in ts 3.5 so you should be able to remove this line.

Reply:
Good call. Addressed in https://github.com/cypress-io/cypress/pull/19601/commits/5ac4525c211aded3fde2db7ce52d7d0b1e405938